ArcGISRuntimeSDKQt_QMLSamples/Scenes/TerrainExaggeration/README.md

```diff
@@ -0,0 +1,22 @@
+# Terrain Exaggeration
+
+This sample demonstrates how to exaggerate a scene's terrain surface.
+
+![](screenshot.png)
+
+## How to use the sample
+Click and drag the slider at the top to change how exaggerated the terrain becomes.
+
+## How it works
+1. Create a scene.
+2. Add a surface feature layer.
```

Comment:
the surface isn't a feature layer. You could say something like "Create and add a surface with an ArcGISTiledElevationService"